includes/class-wp-user-manager.php

```diff
@@ -0,0 +1,488 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-wp-user-manager.php, but found vipgoci-phpcs-scan-iUgbSs.php (*WordPress.Files.FileName.InvalidClassFileName*).